src/relayserver/RelayServer.ts

```diff
@@ -436,50 +439,53 @@ export class RelayServer extends EventEmitter {
     return toBN(await this.contractInteractor.getBalance(this.workerAddress))
   }
 
-  async getShouldRegisterAgain (): Promise<boolean> {
-    const currentBlock = await this.contractInteractor.getBlockNumber()
-    const latestTxBlockNumber = await this._getLatestTxBlockNumber()
+  async getShouldRegisterAgain (currentBlock: number, hubEventsSinceLastScan: EventData[]): Promise<boolean> {
```

Comment:
Please rename to just `shouldRegisterAgain()`

Reply:
Fixed